player.py

```diff
@@ -7,6 +13,67 @@ def __init__(self, Id, strategy):
     def set_strategy(self, new_strategy):
         self.new_strategy = new_strategy
 
+    @staticmethod
+    def generate_strategies(num_of_strategies):
+        """
+        Generate a random order of strategies to give to the players
+        :param num_of_stategies: The number of strategies to pick from
+        :return: list of strategies
+        """
+        strategy_catalogue = ["cities", "settlements", "sheep_monopoly", "wheat_monopoly", "stone_monopoly", "brick_monopoly", "wood_monopoly", "development"]
+        random.shuffle(strategy_catalogue)
+        strategy_list = []
+
+        for i in range(num_of_strategies):
+            choice = strategy_catalogue.pop(0)
+            strategy_list.append(choice)
+
+        return strategy_list
+
     def __repr__(self):
         return "(Name: " + str(self.id) + ", Strategy: " + str(self.strategy) + ")\n"
-    
+
+    @staticmethod
+    def read_strategies(num_of_stategies, file_name):
+        """
+        Read the strategies from a file
+        :param num_of_strategies: The number of strategies to get
+        :param file_name: the name of the file to read from
+        """
+        try:
+            strategies = []
+            with open(file_name) as f:
+                lines = f.readlines()
+                assert(len(lines) == num_of_stategies)
+                for strategy in lines:
+                    strategy = strategy.strip()
+                    if (len(strategy)) > 0 and Player.valid_strategy(strategy):
+                        strategies.append(strategy)
+                    else:
+                        raise ValueError
+
+                return strategies
+        
+        except ValueError:
+            print("Bad value in strategies.txt file...")
+            raise
+
+        except AssertionError:
+            print("strategies.txt must have the same number of strategies as players...")
+            raise
+
+    @staticmethod
+    def valid_strategy(strategy):
+        """
+        Ensure the strategy is valid
+        :param strategy: String, The strategy to validate
+        """
+        strategy_catalogue = ["cities", "settlements", "sheep_monopoly", "wheat_monopoly", "stone_monopoly", "brick_monopoly", "wood_monopoly", "development"]
+        
+        for check_strategy in strategy_catalogue:
+            if strategy == check_strategy:
```

Comment:
Can just do a ternary operator:
`return true if strategy in strategy_catalogue else return false`